orbit/pkg/packaging/windows.go

```diff
@@ -206,15 +214,15 @@ func BuildMSI(opt Options) (string, error) {
 		return "", fmt.Errorf("transform heat: %w", err)
 	}
 
-	if err := wix.Candle(tmpDir, opt.NativeTooling, absWixDir); err != nil {
+	if err := wix.Candle(tmpDir, opt.NativeTooling, absWixDir, opt.Architecture); err != nil {
 		return "", fmt.Errorf("build package: %w", err)
 	}
 
 	if err := wix.Light(tmpDir, opt.NativeTooling, absWixDir); err != nil {
 		return "", fmt.Errorf("build package: %w", err)
 	}
 
-	filename := "fleet-osquery.msi"
+	filename := fmt.Sprintf("fleet-osquery_%s_%s.msi", opt.Version, opt.Architecture)
```

Comment:
We should keep the amd64 one the same name (without prefix).
To not break customer automations that are assuming the name of the output.

Reply:
Some context: https://fleetdm.slack.com/archives/C084F4MKYSJ/p1742477144972289?thread_ts=1742420186.680359&cid=C084F4MKYSJ

Reply:
That seems reasonable, I can save the new name format for the arm release only